### PR TITLE
6 activate event (and 7 inactivate event)

### LINKED ISSUE #6: Activate Events
**Trigger:**

Clicked the Activate Button

**Page:**

Image

**Requirements:**

- [ ] The endpoint should be '/event/activate/{id}'
- [ ] The selected events should be converted to activate

### PR DESCRIPTION
**Pull request by p0tatoes**
- "/event/activate/{id}" and "/event/inactivate/{id}" endpoints can now set an event to "active" or "inactive", respectively
- on successful event activation or inactivation, returns a JSON with:
```
{
     title: event title,
     is_active: boolean,
}
```
- on unsuccessful event activation or inactivation, returns a JSON with:
```
{
     statusCode: 400,
     message: "Unable to {activate or inactivate} event."
}
```

### CODE REVIEW

File: src/event/event.service.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { PrismaService } from 'src/prisma/prisma.service';
+
+@Injectable()
+export class EventService {
+  constructor(private prismaService: PrismaService) {}
+
+  async activateEvent(eventId: number) {
+    const event = this.prismaService.event.update({
+      data: { is_active: true },
+      where: { id: Number(eventId) }, // bruh
```
Comment (rhaeyx) [inline src/event/event.service.ts]: try daw use ug dto for update para didto ang casting. pero okay na man ni, merge na nako

Comment (rhaeyx) [reply src/event/event.service.ts]: if gusto nimo itry with dto para mag implicit casting, ingna lang ko kay di sa nako imerge. pero if di na, kay imerge na nako ni. just lemme know @p0tatoes

Comment (p0tatoes) [reply src/event/event.service.ts]: Noted. I-try nako ug refactor pra mag use ug dto for the `eventId`.

**Comment by p0tatoes** [comment]
@rhaeyx nag use nlng ko ug `ParseIntPipe` para sa route params to make sure na int ang `id` na param sa routes.

**Comment by angellialkuino** [comment]
There were no `await` in the async functions found in `event.service.ts` @rhaeyx @p0tatoes

**Pull request merged**